x-pack/plugins/synthetics/server/legacy_uptime/lib/telemetry/types.ts

```diff
@@ -6,6 +6,16 @@
  */
 import { ServiceLocationErrors } from '../../../../common/runtime_types/monitor_management';
 
+export interface MonitorSyncEvent {
+  totalMonitors: number;
+  totalTestRuns: number;
+  browserTestRunsPer24h: number;
+  httpTestRunsPer24h: number;
+  icmpTestRunsPer24h: number;
+  tcpTestRunsPer24h: number;
```

Comment:
Seems like we should also include known keys, like `browserMonitors` (or just browser, I had mentioned that the `Monitors` postfix is a bit unecessary.